spec/system/employee_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Employee account', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  it 'sign up and sign in' do
+    visit root_path
+    click_link 'Sign Up'
+    fill_in 'Email', with: 'test@test.com'
+    fill_in 'Password', with: 'password'
+    fill_in 'Password confirmation', with: 'password'
```

Comment:
You have created employee factory, and this is the perfect place, to put in in use :) Consider using `attributes_for` method

Reply:
I used a factory, but I couldn't do it with attributes_for.
I understand that using attributes_for speeds up test execution because it doesn't create an object (correct if I'm wrong), but I couldn't use it. In "employee_spec" I left a piece of code that I tried to use attributes_for.

Reply:
You are generally right
attributes_for returns only attributes - no object is created
but there is also build, which is creating an object, that is not saved in db (in contrary to create) - but let's skip it for now - we will get back to that eventually :)

Reply:
Using build - as you did is also fine in this case. 
As I see it `attributes_for` (or `build`) helps to gather all data related to employee instance in one place, and it is easier to later modify it.

I have written more detailed comment in the employee spec - please let me know if it is clear now.